config/config.json

```diff
@@ -15,7 +14,6 @@
   },
   "package_variables": {
       "default_db_username" : "codechecker",
```

Comment:
I think this is also unused.

Reply:
With the new config/run databases and user handling this can be removed too.